### ISSUE TITLE
[BUG] Unable to send images or media files in Desktop App

### ISSUE THREAD

**Issue by alexander-mahisoft** [session-foundation/session-desktop#249]
### Code of conduct

- [X] I have read and agree to adhere to the [Code of Conduct](https://github.com/oxen-io/session-desktop/blob/master/CODE_OF_CONDUCT.md)

### Self-training on how to write a bug report

- [X] I have learned [how to write a bug report](https://www.browserstack.com/guide/how-to-write-a-bug-report)

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I've been unable to send any kind of media files or attachments on private chats or privately created rooms from the Desktop or iOS app since a few days ago.

Media messages get the loading icon for a few seconds and then show "Failed to send"

I'm attaching the Debug log from the Desktop version

### Steps To Reproduce

In a private chat with another user or a private group, send any attachment file

### Desktop Version

v1.11.3

### Anything else?

Debug Log:
[session_debug_1696368503151.txt](https://github.com/oxen-io/session-desktop/files/12797078/session_debug_1696368503151.txt)

Example screenshot:
![image](https://github.com/session-foundation/session-desktop/assets/84341320/c52f51cd-c7b6-489f-81e3-31e61193e00d)

**Comment by Bilb** [comment]
Yeah this is an unrelated issue then. We just need better support for communities and their different permissions
